core-errors/src/lib.rs

```diff
@@ -159,6 +159,10 @@ pub enum MessageError {
     /// An error occurs in attempt to charge gas for dispatch stash hold.
     #[display(fmt = "Not enough gas to hold dispatch message")]
     InsufficientGasForDelayedSending = 399,
+
+    /// An error when a syscall is used incorrectly
+    #[display(fmt = "Syscall was used incorrectly")]
```

Comment:
Why do you need this error?

Reply:
It was an error that already existed that was being used if the programid made was the system id, and I just renamed it.

Reply:
can be removed now, see our conversation above